html/comparison.html.haml

```diff
@@ -619,8 +611,7 @@
                 %span.summary-part{"ng-show" => "diff.services.only_in2"}
                   {{diff.meta.description_b}}: {{diff.services.only_in2.services.length || 0}} services ({{diff.services.only_in2.init_system}})
                 %span.summary-part{"ng-show" => "diff.services.common"}
-                  %a{:class=>"show-common-elements", href: "#services_both"}
-                    both
+                  %a{ :class => "show-common-elements", href: "#services_both" }both
```

Comment:
Line is too long. [84/80]<br>Avoid defining `class` in attributes hash for static class names